sormas-backend/src/test/resources/META-INF/persistence.xml

```diff
@@ -101,7 +103,7 @@
 	    	<property name="javax.persistence.jdbc.password" value="" />
 
 			<!-- Database schema is created before each test by TestDatabaseCleaner class -->
-      		<property name="hibernate.hbm2ddl.auto" value="none" />
+      		<!--<property name="hibernate.hbm2ddl.auto" value="create"/>-->
```

Comment:
Replace code by `<!-- Database schema is freshly created before each test by TestDatabaseCleaner -->`